src/__tests__/general.test.ts

```diff
@@ -37,6 +37,30 @@ describe("General API", () => {
 
     const orgList = await generalAPI.listOrganizations();
 
-    expect(Array.isArray(orgList.data)).toBe(true);
-  }, 10000);
+        expect(Array.isArray(orgList.data)).toBe(true);
+
+    },10000);
+
+    it('Should list down all the stations',async () => {
+        let generalAPI = new GeneralApi(configuration);
+        const StationList = await generalAPI.listStations();
+        expect(Array.isArray(StationList.data)).toBe(true);
+        
+    },10000);
+
+    it('Should get status', async () => {
+        let generalAPI = new GeneralApi(configuration);
+        const status = await generalAPI.getStatus('');
```

Comment:
for /status api , tdei_record_id is mandatory parameter. This might be case for negative testing, please handle response assertion accordingly

Reply:
done